### ISSUE TITLE
Questions about ChebConv

### ISSUE THREAD

**Issue by JYeonLee** [rusty1s/pytorch_geometric#901]
Hello, I have two questions about ChebConv.

First,
I read torch_geometric.nn.ChebConv class. When I read the forward function, I don't understand using lambda_max of 2.0 in code as follows:
```    
def forward(self, x, edge_index, edge_weight=None, batch=None,
                lambda_max=None):
        ...
        lambda_max = 2.0 if lambda_max is None else lambda_max

        edge_index, norm = self.norm(edge_index, x.size(0), edge_weight,
                                     self.normalization, lambda_max,
                                     dtype=x.dtype, batch=batch) 
        ...
```

As far as I know, when calculating lambda_max, ``` torch_geometric.transforms.LaplacianLambdaMax() ``` function is used. However, in code, lambda_max just sets 2.0 if lambda_max is none.  Also, as far as I know, setting lambda_max  = 2 is Kipf's GCN, not ChebConv using Chebyshev polynomial. Why should we use lamda_max as 2?

Second,
I am trying to batch training for multiple graphs. In the documentation of CheConv, I read that ```lambda_max``` should be a ```torch.Tensor``` of size ```[num_graphs]``` in a mini-batch scenario. This means that the list is [lambda_max_1, ... , lamda_max_n] where n is the number of graphs. Is that correct?

**Comment by rusty1s** [comment]
1. `lambda_max=2` corresponds to using the normalized Laplacian, where the largest eigenvalue can not be greater than `2`. This is not GCN and instead formalized in the Cheby paper. GCN makes use of this though.
2. That is correct!

**Comment by JYeonLee** [comment]
Thanks a lot for your reply. I understood.

**Comment by JYeonLee** [comment]
I have another question.

Then, what is the range of edge_weight which is norm function's return value in ChebConv?
```     def norm(edge_index, num_nodes, edge_weight, normalization, lambda_max,
             dtype=None, batch=None):

        edge_index, edge_weight = remove_self_loops(edge_index, edge_weight)

        edge_index, edge_weight = get_laplacian(
            edge_index, edge_weight, normalization, dtype, num_nodes)

        if batch is not None and torch.is_tensor(lambda_max):
            lambda_max = lambda_max[batch[edge_index[0]]]

        edge_weight = (2.0 * edge_weight) / lambda_max
        edge_weight[edge_weight == float('inf')] = 0

        edge_index, edge_weight = add_self_loops(
            edge_index, edge_weight, fill_value=-1, num_nodes=num_nodes)

        return edge_index, edge_weight
```
When I use lamda_max as 2, the range is [-1,1].
Because in my case, the range is [-1, 1.5518]. So I am very confused.
